auth-api/tests/unit/api/test_task.py

```diff
@@ -156,3 +156,15 @@ def test_put_task_product(client, jwt, session, keycloak_mock):  # pylint:disabl
     dictionary = json.loads(rv.data)
     assert rv.status_code == http_status.HTTP_200_OK
     assert dictionary['status'] == TaskStatus.COMPLETED.value
+
+
+def test_fetch_task(client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that the task can be fetched by id."""
+    user = factory_user_model()
+    task = factory_task_service(user.id)
+    task_id = task._model.id
+
+    headers = factory_auth_header(jwt=jwt, claims=TestJwtClaims.staff_role)
+    rv = client.get('/api/v1/tasks/{}'.format(task_id), headers=headers, content_type='application/json')
+    assert rv.status_code == http_status.HTTP_200_OK
+    assert rv.json.get('name') == task._model.name
```

Comment:
not in this PR ;but can you write a big test case for asserting org status and mail notification when task is completed... like a full workflow... create org ; make sure task is getting created ; update task ;make sure org is udpated and mail is being sent [mock the mail code and check number of invocation]... this is low priority..but such a test case will be really helpful

Reply:
It is already there with the previous PR :)